pom.xml

```diff
@@ -226,6 +226,12 @@
                 <groupId>org.wildfly</groupId>
                 <artifactId>wildfly-arquillian-container-managed</artifactId>
                 <version>${wildfly.version}</version>
+                <exclusions>
```

Comment:
It showed up in Eclipse for me. Not in the CLI build. Hence I added it with this issue key.